### PR TITLE
manage studies

### PR DESCRIPTION
**Pull request by AbdelHedhili**
Signed-off-by: Abdelsalem <abdelsalem.hedhili@rte-france.com>

### CODE REVIEW

File: src/utils/rest-api.js [hunk 1]
```diff
@@ -119,11 +119,97 @@ export function deleteElement(elementUuid) {
...
+        changeStudyAccessRightsUrl = getStudyUrl(studyUuid) + '/private';
+    } else {
+        changeStudyAccessRightsUrl = getStudyUrl(studyUuid) + '/public';
+    }
+
+    console.debug(changeStudyAccessRightsUrl);
+    return backendFetch(changeStudyAccessRightsUrl, {
+        method: 'POST',
+        headers: {
+            Accept: 'application/json',
```
Comment (ChamseddineBhd) [inline src/utils/rest-api.js]: delete those headers !

File: src/components/directory-tree-view.js [hunk 2]
```diff
@@ -308,7 +308,7 @@ const DirectoryTreeView = ({ rootDirectory }) => {
         ws.onmessage = function (event) {
             if (isConcerned()) {
                 displayErrorIfExist(event);
-                updateDirectoryChildren(selectedDirectoryRef.current, false);
+                updateDirectoryChildren(selectedDirectoryRef.current);
```
Comment (ChamseddineBhd) [inline src/components/directory-tree-view.js:311]: why remove false ?

File: src/components/directory-content.js [hunk 3]
```diff
@@ -46,10 +65,181 @@ const DirectoryContent = () => {
...
+                        })
+                    );
+                } else {
+                    handleCloseStudyAccessRights();
+                }
+            }
+        );
+    };
+
+    const StyledMenu = withStyles({
```
Comment (ChamseddineBhd) [inline src/components/directory-content.js]: why defined here in the component?

**Review by ChamseddineBhd — CHANGES_REQUESTED** [review]

File: src/components/directory-tree-view.js (hunk 2 above)
Comment (AbdelHedhili) [reply src/components/directory-tree-view.js:311]: because your method only takes one parameters

**Review by ChamseddineBhd — APPROVED** [review]

**Pull request merged**